viaq-jenkins/fields.yml

```diff
@@ -0,0 +1,493 @@
+version: 2016.03.21.0
+
+defaults:
+  type: string
+
+
+_default_:
+  type: group
+  description: >
+    Contains common fields, corresponds to mappings of the _default_.
+    This mapping represents generic Jenkins configuration.
+  fields:
+    - name: "@timestamp"
+      type: date
+      format: yyyy-MM-dd'T'HH:mm:ss.SSSSSSZ||yyyy-MM-dd'T'HH:mm:ssZ||dateOptionalTime
+      example: 2015-01-24T14:06:05.071Z
+      description: >
+        UTC value marking when the log payload was created, or when log payload was first collected if the creation time is not known; 
```

Comment:
it'll be converted to a UTC value.
value could also be in specific timezone, but it must be convertible to UTC.